publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/config/SecurityConfig.java

```diff
@@ -74,20 +105,54 @@ public Integer getTimeOut() {
     @Autowired
     private CustomAuthenticationEntryPoint customAuthenticationEntryPoint;
 
-    @Autowired
-    protected void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
+    @Bean
+    public UserCache userCache() {
+        // Adjust cache settings as necessary
+        return new SpringCacheBasedUserCache(cacheManager().getCache("authenticationCache"));
```

Comment:
Should always a new instance be created? Is it just a wrapper?

Reply:
refactored to use a single isntance